bin/createsrcdirandfiles.js

```diff
@@ -1,169 +1,228 @@
 /**
  * creating src directories and files
- * @param {*} details 
+ * @param {*} details
  */
 const fs = require("fs");
 
 const openAppendFile = require("./openandappendfile");
 const {
-    controllers,
-    userController,
-    userModelData,
-    noSequelizeSetupData,
-    sequelizeSetupData,
-    middleware,
-    userMiddleware,
-    noSequelizeUserModelData,
-    routes,
-    userRouter,
-    appJsTest,
-    createDb,
-    dropDb,
-    noOrmcreateDb,
-    noOrmDropDb,
-    userQueries,
-    appJs,
-    noOrmUserController
+  controllers,
+  userController,
+  userModelData,
+  noSequelizeSetupData,
+  sequelizeSetupData,
+  middleware,
+  userMiddleware,
+  noSequelizeUserModelData,
+  routes,
+  userRouter,
+  routeTest,
+  middlewareTest,
+  useControllerTest,
+  createDb,
+  dropDb,
+  noOrmcreateDb,
+  noOrmDropDb,
+  userQueries,
+  appJs,
+  noOrmUserController
 } = require("../tasks");
 
-const createSrcDirAndFiles = (details) => {
-    const { appBaseDirectory, tests, database, orm } = details;
-    let foldersToAdd = ['src/controllers', 'src/routes', 'src/config', 'src/scripts', 'src/models', 'src/middlewares', 'src'];
-    if (tests) {
-        foldersToAdd.push('test');
+const createSrcDirAndFiles = details => {
+  const { appBaseDirectory, tests, database, orm } = details;
+  let foldersToAdd = [
+    "src/controllers",
```

Comment:
I think this is good and readable.

Reply:
@ezkemboi  Thanks, I think we should set up linting for this project, I could create an issue addressing this.

Reply:
https://github.com/kemboijs/nodejs-api-cli/issues/25